cmd/scollector/collectors/apache_mod_info.go

```diff
@@ -0,0 +1,121 @@
+package collectors
+
+import (
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+
+	"golang.org/x/net/html"
+
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+)
+
+func init() {
+	collectors = append(collectors, &IntervalCollector{F: c_apache_mod_info, Interval: time.Minute * 30})
+}
+
+// Extract from HTML recieved from http://127.0.0.1/server-info?server
+// Find the line that looks like:
+// "Timeouts: connection: 60 keep-alive: 15"
+// Returns the two values contained therein.
+
+func extract_timeouts(doc *html.Node) (int, int) {
```

Comment:
See http://golang.org/doc/effective_go.html#commentary for how to do go-style comments. Short version: remove line 25 and make the first word extract_timeouts.